### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by crydotsnake**
It is always better, to download it from the official Website :).

### CODE REVIEW

**Comment by cclauss** [comment]
My sense is that installing from Microsoft's store is a better way to install the current version of Python is a way where it will be easily updated as Python progresses.

**Comment by richardlau** [comment]
Maybe instead of replacing we could list both options?

File: README.md [hunk 1]
```diff
@@ -43,7 +43,7 @@ Depending on your operating system, you will need to install:
 
 ### On Windows
 
-Install the current version of Python from the [Microsoft Store package](https://docs.python.org/3/using/windows.html#the-microsoft-store-package).
+Install the current version of Python from the [Python Website](https://www.python.org/downloads/windows/).
```
Comment (richardlau) [inline README.md]: Maybe instead of replacing we could list both options?

Comment (cclauss) [reply README.md]: I do not see the advantage of selecting the python.org option but I do see the downside of selecting it.

Comment (crydotsnake) [reply README.md]: I agree with @richardlau

**Pull request closed without merging**